.gitignore

```diff
@@ -37,4 +37,5 @@ DerivedData
 
 venv/
 
-.vscode/
\ No newline at end of file
+.vscode/
+*.profraw
```

Comment:
Why is this necessary? :)

Reply:
Because `.vscode/` is the folder where the Visual Studio Code saves the files of the editor. I think we shouldn't commit personal config files to the repo. Note that this prevents adding the folder .vscode when you `git add .`, but we can add a file with `--force` if needed (for example recommending and extension, or doing some build task, and so on)
`*.profraw` files are the result of doing coverage tests. [Elkiwa left a great guide on discord](https://discord.com/channels/884163483409731584/1013902234469269554/1186670417360977921)